Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -1596,20 +1597,21 @@ class KeyboardViewController: UIInputViewController {
         }
         if DeviceType.isPad
           && key == "@"
+          && !usingExpandedKeyboard
           && (controllerLanguage == "Portuguese"
             || controllerLanguage == "Italian"
             || commandState == .translate)
         {
           leftPadding = keyWidth / 3
           addPadding(to: stackView1, width: leftPadding, key: "@")
         }
-        if DeviceType.isPad
+       /* if DeviceType.isPad
           && key == "$"
           && controllerLanguage == "Italian"
         {
           leftPadding = keyWidth / 3
           addPadding(to: stackView1, width: leftPadding, key: "$")
-        }
+        }*/
```

Comment:
nit - We OK to just remove the newly commented code?

Reply:
Yeah, It would be OK, I commented it out realizing there were better ways